unittests/fixtures/factory/work-dir/entry_el7_osg34/submit_attrs.cfg

```diff
@@ -0,0 +1,4 @@
+# File: submit_attrs.cfg
```

Comment:
I see that one of the directory names in the file path says “entry_el7_osg34". I only know osg36 and osg35. Is this something new?

Reply:
This is old. OK for now, we should update sometime